### ISSUE TITLE
Any plans to support DALI?

### ISSUE THREAD

**Issue by mitming** [open-mmlab/mmcv#1690]
For some tasks (e.g. contrastive learning), data augmentation is very important and data augmentation on CPU is often very slow, are there any plans to support the use of DALI in the future?

**Comment by zhouzaida** [comment]
Yes, we've discussed data processing acceleration before, but we think it is not a high-priority plan. However, if you have any plans, feel free to create PR.

**Comment by mitming** [comment]
> Yes, we've discussed data processing acceleration before, but we think it is not a high-priority plan. However, if you have any plans, feel free to create PR.

Hi @zhouzaida ,

Maybe [this repo](https://github.com/tanglang96/DataLoaders_DALI) could be implemented in mmcv with DALI?

**Comment by zhouzaida** [comment]
Thank you for the reference, we will look into it.
